webapp/src/components/EstateDetailPage/EstateDetail.js

```diff
@@ -53,13 +53,16 @@ export default class EstateDetail extends React.PureComponent {
               </Grid.Column>
               {isOwner && (
                 <Grid.Column className="parcel-actions-container" computer={8}>
-                  <EstateActions onEditMetadata={onEditMetadata} />
+                  <EstateActions
+                    onEditMetadata={onEditMetadata}
+                    assetId={estate.asset_id}
+                  />
                 </Grid.Column>
               )}
               {allParcels && (
                 <React.Fragment>
                   <Grid.Column
-                    width={8}
+                    width={isOwner ? 8 : 16}
```

Comment:
This is kind of weird. Maybe abstracting into a constant? naming floating numbers it's (almost) always a good idea